### ISSUE TITLE
Missing CA2000

### ISSUE THREAD

**Issue by SixFive7** [dotnet/roslyn-analyzers#2874]
#### Analyzer package
Microsoft.CodeAnalysis.FxCopAnalyzers

#### Package Version
v2.9.5-beta1.final (Latest)

#### Diagnostic ID
[CA2000](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2000-dispose-objects-before-losing-scope?view=vs-2019)

#### Repro steps

```csharp
using System;
using System.Net.WebSockets;
using System.Threading;

namespace CA2000
{
    class Program
    {
        static void Main()
        {
            var CancellationTokenSource = new CancellationTokenSource();
            var ClientWebSocket = new ClientWebSocket();
        }
    }
}
```

#### Expected behavior

Clearing the two CA2000 warnings by changing both lines:
```csharp
            using var CancellationTokenSource = new CancellationTokenSource();
            using var ClientWebSocket = new ClientWebSocket();
```

#### Actual behavior

Changing only one of two lines, instead of both, clears both warnings.

**Comment by mavasani** [comment]
We bail out in presence of any using var declaration due to missing IOperation support for using declarations. Tagging @chsienki

**Comment by mavasani** [comment]
This is fixed with VS2019 16.4
